app/routes/router.js

```diff
@@ -92,6 +92,16 @@ router
     .get('/findalltransactionsbyparamid', function (req, res){
         API.findTransactionsByParamId(req, res)
     })
+	
+	// gets all the errors
+    .get('/findallerrorreports', function (req, res){
```

Comment:
Can we change this endpoint to just ```/errors```? These long endpoint names are overly verbose and the action is defined by the HTTP method.